### PR TITLE
refactor(app): adding id to collapsible step

### PR DESCRIPTION
**Pull request by reachojhaneha**
adding the id for the collapsible step, as this is need in "NO MODULE" testcase.
I want to verify that step 2 is Labware setup and not Module setup.

### CODE REVIEW

File: app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ export function CollapsibleStep({
             color={C_MED_GRAY}
             fontWeight={FONT_WEIGHT_SEMIBOLD}
             fontSize={FONT_SIZE_BODY_1}
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx:47]: Oof, it already looks like we have an id prop thats getting passed into this component, which is very confusing. Usually if we pass an id into a child component, that id should be assigned to the top level DOM element (in this case the `Flex`). Rn it's being assigned to the `title` text deeper in the tree, so we should probably remove the id prop from this component and instead construct our own ids from inside of this component and pass them into the title text directly.

For the three text elements we can just have: 

```
 id={`CollapsibleStep_${label}`}
 id={`CollapsibleStep_${title}`}
 id={`CollapsibleStep_${description}`}
```

Does that make sense?

Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx:47]: yes, it makes sense. As far as the ids for testing is concerned
 id={`CollapsibleStep_${label}`} - will give me the text for "STEP1 /2/3"
id={`CollapsibleStep_${title}`} == "id=RunSetupCard_robot_calibration_step"(this id is already available)
id={`CollapsibleStep_${description}`} -> i don't need this id for my testcase. As i am not verifying the paragraph content.

Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx:47]: however, if you feel i should remove id prop from title and assign it inside the component. I am happy to do so.

Comment (shlokamin) [reply app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx:47]: >  if you feel i should remove id prop from title and assign it inside the component. I am happy to do so.

that'd be awesome, thank you!

Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx:47]: this is done

**Review by shlokamin — APPROVED** [review]
approved pending CI —looks like you still need a semantic PR title though. you can use commitzen to do this for you.

**Pull request merged**